### PR TITLE
[test] Remove max version limits on studio analytics test

### PR DESCRIPTION
**Pull request by saimidu**
*GitHub Issue #, if available:*

Note: If merging this PR should also close the associated Issue, please also add that Issue # to the Linked Issues section on the right. 

### Description
Remove maximum version limit on SM Studio Analytics package presence test

## Label Checklist
- [x] I have added the project label for this PR (*<project_name>* or "Improvement")

## PR Checklist
- [x] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron/graviton] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]
- [x] If the PR changes affects SM test, I've modified dlc_developer_config.toml in my PR branch by setting sagemaker_tests = true and efa_tests = true
- [x] If this PR changes existing code, the change fully backward compatible with pre-existing code. (Non backward-compatible changes need special approval.)
- [ ] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [ ] (If applicable) I've documented below the tests I've run on the DLC image
- [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: test/dlc_tests/sanity/test_utility_installation.py [hunk 1]
```diff
@@ -129,13 +129,10 @@ def test_apache_tomcat(image):
...
 def test_sagemaker_studio_analytics_extension(training, package_name):
     framework, framework_version = test_utils.get_framework_and_version_from_tag(training)
-    utility_package_minimum_framework_version = {"pytorch": "1.7", "tensorflow": "2.4"}
-    utility_package_maximum_framework_version = {"pytorch": "1.8", "tensorflow": "2.6"}
-    
-    if framework not in utility_package_minimum_framework_version or Version(framework_version) < Version(
-        utility_package_minimum_framework_version[framework]) or Version(framework_version) > Version(
-        utility_package_maximum_framework_version[framework]
-    ):
+    studio_analytics_framework_version_limits = {"pytorch": ">=1.7", "tensorflow": ">=2.4"}
```
Comment (arjkesh) [inline test/dlc_tests/sanity/test_utility_installation.py:132]: does this also need to run for huggingface? I believe that function "get_framework_and_version_from_tag" will return "huggingface_pytorch" and "huggingface_tensorflow" for those cases

File: pytorch/training/docker/1.11/py3/cu115/Dockerfile.gpu [hunk 2]
```diff
@@ -480,6 +480,13 @@ RUN pip install --no-cache-dir -U \
  && pip uninstall -y torch \
  && pip install --no-cache-dir -U ${PT_TRAINING_URL}
 
+# Install Studio Notebook Utils
```
Comment (arjkesh) [inline pytorch/training/docker/1.11/py3/cu115/Dockerfile.gpu:483]: have you tested that the cu115 images build successfully? i don't think they are enabled in the buildspec

**Comment by arjkesh** [comment]
@saimidu please also include a description of the dockerfile changes made - right now this PR only references the test
